css-basics/css-basics_pokemon-page/css/styles.css

```diff
@@ -5,11 +5,19 @@
 /* Task 1: First of all we want to adjust how the size of our elements is calculated when working with CSS. Do you know how to switch from "content-box" to "border-box"? */
 
 /* your solution */
+*{
+  * {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+  }
+}
+
```

Comment:
You are nesting 2 universal selectors. I don't think that is necessary (does it even work?).